conference/cfp/models.py

```diff
@@ -31,21 +31,39 @@ class Submission(models.Model):
     updated_at = models.DateTimeField(
         _('Updated'), auto_now=True
     )
+    notes = models.TextField(_('Notes'))
```

Comment:
I'll add blank=null, because Django is saving empty TextField as empty string '' not None/null.